### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by demeringo** [Boavizta/cloud-scanner#524] [Announcements]
We just released a v3.0.0 of cloud scanner.

The main evolution is the ability to simulate the impacts of an arbitrary inventory. This inventory can be passed as json (Command line or Web API).

> [!IMPORTANT]
> This release introduces breaking changes in the format of the json inventory, as well as command line options.

- Changelog: https://github.com/Boavizta/cloud-scanner/blob/main/CHANGELOG.md
- Releases: https://github.com/Boavizta/cloud-scanner/releases

If you have the opportunity to test this version, please do not hesitate to provide feedback here, or by openning an issue ;-)

**Comment by damienfernandes** [comment]
@demeringo Hello, I tried feature inventory of new version. It's work well ! Some breaking changes as annonce but easy to fix ;-)
